plugin-commons/src/test/java/org/sonarsource/sonarlint/core/plugin/commons/loading/PluginClassloaderFactoryTest.java

```diff
@@ -32,7 +31,7 @@
 import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class PluginClassloaderFactoryTest {
+class PluginClassloaderFactoryTest {
```

Comment:
```suggestion
class PluginClassloaderFactoryTests {
```

Reply:
We sometimes use plural sometimes not, what should be the rule ? Do you have any recommended naming convention in mind ?

Reply:
I think JUnit 5 changed to use plural and personally I find it better.
So the rule is:
* JUnit4 = singular: https://github.com/junit-team/junit4/wiki/Getting-started
* JUnit 5 = plural: https://junit.org/junit5/docs/current/user-guide/#writing-tests